### ISSUE TITLE
Excessive delay in environment checking on startup

### ISSUE THREAD

**Issue by ajcasagrande** [edgexfoundry/edgex-ui-go#534]
# 🐞 Bug Report

### Affected Services [**REQUIRED**]
edgex-ui

### Is this a regression?
no

### Description and Minimal Reproduction [**REQUIRED**]

Unless this is some kind of DDOS protection, the web-ui has way too many calls to sleep for 1.5 seconds at a time. It took me 5.2 seconds to load the page, and 4.9 seconds of that is "IDLE" time.

Just look at this:

```ts
   environmentCheck() {
    this.progressMsg = "environment checking";
    window.setTimeout(()=>{
...
        if (resp === "secure") {
...
        } else {
...
          window.setTimeout(()=>{
            this.navigateToHome();
          },1500)
        }
      });
    },1500)
  }

  navigateToHome() {
    window.setTimeout(()=>{
      this.router.navigate(['/dashboard'], { relativeTo: this.route })
    },1500)
  }
```
That is 4.5 seconds of sleep just there.

## 🔥 Exception or Error

![Screenshot from 2022-07-06 15-15-05](https://user-images.githubusercontent.com/48735240/177652575-1b45eed1-0b1c-4f2f-9aa4-79a43c70380b.png)

## 🌍  Your Environment
NON-Secure Mode
Latest main branch as of time of submit issue

**Comment by badboy-huaqiao** [comment]
@ajcasagrande yes, good catch, I found that users have a not good usage habit, they always cilck the refresh button of the brower no matter what problems they run into, they think it will fix the troubles through the refresh button of the browser, it will be useful to the legacy web app, but it doesn't make any sense in a SPA web.

anyther reason is because GUI must support both secure and insecure in one SPA web, maybe user will change the EdgeX from insecure to secure environment without reopening the GUI on browser, so GUI must check the environment at any time, I don't know how and what users do for their environment , but what I can do is that restrict it on the GUI.

because it's a local GUI, not running in the cloud, there is almost no network latency time when environment changed between secure and insecure mode,so leave a sleep time will give user a better `User Experience` to let user see the transition state.

it's ok let's remove the sleep time now, maybe we'll figure out some better solutions supportting the above feature, like adding a secure or insecure mode tag on the GUI.

the version tags also should be needed.
I'm so glad work with you, I saw your PRs, I believe we will build a better GUI for EdgeX together.

by the way, I' working on refactoring the metadata module, splitting the big component into some small components, so  we can removing the reduntant code, and easer to maintain, it will be done next week, I will invite you to review it together.
